chart/values.schema.json

```
@@ -1737,6 +1737,23 @@
                         }
                     }
                 },
+                "resources": {
+                    "description": "Resources for the migrate database job pod",
+                    "type": "object",
```

We should specify the exact type - `ResourceRequirements`.
Example:
https://github.com/artifacthub/hub/blob/7dc1946d8ce108844121b33930cb2fd2b036c0e8/charts/artifact-hub/values.schema.json#L231-L236

---

```suggestion
                    "type": "object",
                    "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.19.0/_definitions.json#/definitions/io.k8s.api.core.v1.ResourceRequirements",
```

You mean like this? Sorry, but are you sure? Just want to double check because I don't see even a single mention of `$ref` in this file. Please let me know if it's indeed the case or just click on "commit suggestion".